carbon-design-system/carbon-addons-iot-react
# fix(imagecard): set of small fixes to imagecard and the dashboardeditor

Closes #

**Summary**

- Small set of fixes to the Image Card and default settings for the Image Card inside the Dashboard Editor

**Change List (commits, features, bugs, etc)**

- fix(editorUtils): default the displayOption to 'contain' as the card works better that way (and hotspots are located in the correct place)
- fix(ImageHotspots): don't call the callback if it's not set (it's not set if you click without pressing CTRL)
- fix(ImageHotspots): have to put the index in the key because you can have two hotspots with the same x and y value and it will throw an console error

**Acceptance Test (how to verify the PR)**

- Verify adding new Image Cards inside the DashboardEditor default their Setting for displayOption correctly
- Verify that if you Edit the Image in the Dashboard Editor story, switch to the Labels tab and then click the image without pressing CTRL that no error is logged to the console
- Verify that the new 'text' hotspot shows in all the existing ImageHotspots stories

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify that fixed hotspots render correctly in the existing stories
